pkg/hintrunner/utils/math_utils.go

```diff
@@ -26,6 +26,24 @@ func EcDoubleSlope(pointX, pointY, alpha, prime *big.Int) (big.Int, error) {
 	return Divmod(n, m, prime)
 }
 
+func LineSlope(point_aX, point_aY, point_bX, point_bY, prime *big.Int) (big.Int, error) {
+	// https://github.com/lambdaclass/cairo-vm_in_go/blob/31c3628bc10ebc1628685b3cdfa72d0e938b533e/pkg/builtins/ec_op.go#L258
+
+	modValue := new(big.Int).Mod(new(big.Int).Sub(point_aX, point_aY), prime)
```

Comment:
I think the subtraction here is meant to be done between `point_aX` and `point_bX` according to this [line](https://github.com/starkware-libs/cairo-lang/blob/efa9648f57568aad8f8a13fbf027d2de7c63c2c0/src/starkware/python/math_utils.py#L135C5-L135C44):
`assert (point1[0] - point2[0]) % p != 0`